feat: add keyboard shortcut to move selected nodes (unbound by default)

duckcomfy opened the pull request:
closes #1265

## Summary

This PR adds four new keyboard shortcuts ~~(CTRL + Up/Down/Left/Right Arrow Key)~~ (unbound by default) which allow moving selected nodes by a set number of pixels, configured by the `Comfy.SnapToGrid.GridSize` setting.

## Video demonstration

[comfy_ctrl_arrow-2025-06-03_21.35.51.webm](https://github.com/user-attachments/assets/074c6568-8aa2-4355-aae8-2f4f9e9f568f)

## The shortcuts in the shortcut settings

![1748979403](https://github.com/user-attachments/assets/b6088699-f717-4a0c-a4bc-d367c9a4b1e0)

┆Issue is synchronized with this [Notion page](https://www.notion.so/PR-4066-feat-add-keyboard-shortcut-to-move-selected-nodes-with-CTRL-ArrowKey-2076d73d365081dca12bc0c2fb2b9802) by [Unito](https://www.unito.io)

Commit:
feat: add keyboard shortcut to move selected nodes with CTRL+ArrowKey

closes #1265

Unit tests included.

Please note that I had to add some `void` operators to unrelated code in order for Husky to accept my commit; this should not have any functional change.

Ensure `moveSelectedNodesVersionAdded` is correct before merging this commit's associated PR.

webfiltered reviewed (changes requested):
Hi, looks good.  I agree w/ @christian-byrne - we should not add default key bindings for this feature.

duckcomfy commented on browser_tests/fixtures/ComfyPage.ts:
```diff
@@ -762,7 +762,7 @@ export class ComfyPage {
       }
     })
-    this.page.mouse.move(10, 10)
+    void this.page.mouse.move(10, 10)
```
As long as you're sure this is safe - this is not my code, so awaiting this previously unawaited promise could cause regressions.

Commit:
fix: do not register default keybinds for moving selected nodes

Let user set their own, otherwise the experience might be too jarring to start happening suddenly.

refactor: await the disregarded promise instead of voiding it (code unrelated to this PR, but the change was required by ESLint.)

ci: fix tests to use the commands rather than the keybinds, since I removed the default keybinds

duckcomfy commented:
Thank you for the quick review, I've changed the PR so that the commands are registered without default keybinds in [77a447c](https://github.com/Comfy-Org/ComfyUI_frontend/pull/4066/commits/77a447c9a002f85c98b79ca1ded12b7b7897451b)

duckcomfy commented:
I've fixed the test and this PR is now ready for final review.

webfiltered reviewed (changes requested):
LGTM - will merge once the minor nits are in.  ... and thank you!

duckcomfy committed:
Update browser_tests/fixtures/ComfyPage.ts

remove ESLint-mandated use of `void` operator

Co-authored-by: filtered <176114999+webfiltered@users.noreply.github.com>

duckcomfy committed:
bump new command's version number

Co-authored-by: filtered <176114999+webfiltered@users.noreply.github.com>

christian-byrne commented:
In the future let's tag these types of PRs (UI/UX, product) with `Design` label
